index.html

```diff
@@ -1,9 +1,108 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<head>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+<header>
+    <div>
+      <div>
+        <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+        <span> Nifty Penguin Magic </span>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="#"> npm Enterprise </a></li>
+          <li><a href="#"> Products </a></li>
+          <li><a href="#"> Solutions </a></li>
+          <li><a href="#"> Resources </a></li>
+          <li><a href="#"> Docs </a></li>
+          <li><a href="#"> Support </a></li>
+        </ul>
+      </nav>
+    </div>
+  
+    <div>
+        <div>
+            <img class="npmLogo" src="images/npm-logo.png" alt="npm logo" />
+          </div>
+          <div>
+            <img class="magnifyingGlass" src="images/magnifying-glass.png" alt="magnifying glass" />
+          </div>
+          <div class="searchBar">
+            <input type="text" placeholder="Search packages"><input type="button" value="Search" />
```

Comment:
You can use input for the search input and for the button the tag <button></button>. This way you can manage in CSS both fields in a different way. You can also add classes to every input and this way you can manage them separately too.